ui/lib/core/addon/components/edit-form.js

```diff
@@ -42,20 +42,21 @@ export default Component.extend({
       }
       return;
     }
-    this.get('flashMessages').success(this.get(messageKey));
+    this.flashMessages.success(this.get(messageKey));
```

Comment:
can the `this.get` be removed in favor of `this.messageKey` here as well?

Reply:
If the key we were accessing were called `messageKey` we could do that here, but `messageKey` is a local var that references `successMessage` or `deleteSuccessMessage` here: https://github.com/hashicorp/vault/blob/c9c91e66cc6f0a2e079584abb81c28aeabb180ae/ui/lib/core/addon/components/edit-form.js#L34. Doing it this way lets us swap between the two with the single call.